intranet/templates/bus/morning.html

```diff
@@ -0,0 +1,62 @@
+{% extends "page_with_nav.html" %}
+{% load static %}
+{% load svg %}
+{% load pipeline %}
+
+{% block title %}
+    {{ block.super }} - Bus
+{% endblock %}
+
+{% block css %}
+    <link rel="stylesheet" href="{% static 'vendor/selectize.js-0.12.4/dist/css/selectize.default.css' %}">
+    {{ block.super }}
+    {% stylesheet 'bus' %}
+    {% stylesheet 'polls' %}
+    {% stylesheet 'dashboard' %}
+{% endblock %}
+
+{% block js %}
+    {{ block.super }}
+    {% include "bus/bus_head.html" %}
+{% endblock %}
+
+{% block head %}
+    {% if dark_mode_enabled %}
+        {% stylesheet 'dark/base' %}
+        {% stylesheet 'dark/nav' %}
+        {% stylesheet 'dark/bus' %}
+    {% endif %}
+{% endblock %}
+
+{% block main %}
+    <div class="primary-content">
+        <h2 class="welcome-text">Good morning!</h2>
+        {% if admin %}
+            <h2 class="title">Mark bus:</h2>
+            <select id="buses">
+                {% for bus in bus_list %}
+                    <option value="{{ bus.route_name }}">{{ bus.route_name }}</option>
+                {% endfor %}
+            </select>
+            <div class="update-button">
+                <button onclick="sendMorningUpdate({'status':'a'})">Arrived</button>
```

Comment:
This feature is broken (i.e. I cannot mark buses)

Reply:
First issue there is actually related to the second...in bus.js there is an additional if statement that only makes the morning bus login work before noon. I'm going to put that into a variable in settings so the hardcoding won't be an issue in the future.

Reply:
Okay, try now. I removed all the hardcoding of the hour and configured it in settings. To debug after noon, you will have to change it to 22 or something. As a note, changing it to 24 makes celery mad, but this doesn't seem like a reasonable use case so I didn't really worry about it.